platform/darwin/src/MGLStyle.h

```diff
@@ -189,7 +189,7 @@ static const NSInteger MGLStyleDefaultVersion = 9;
 /**
  A set containing the style’s sources.
  */
-@property (nonatomic, strong) NS_MUTABLE_SET_OF(MGLSource *) *sources;
+@property (nonatomic, strong) NS_SET_OF(__kindof MGLSource *) *sources;
```

Comment:
`-mutableValueForKey:` is available to all KVC-compliant properties. The fact that it happens to be optimized for these two properties is an implementation detail. But if we can think of a good reason to use it in an example, I'd be all for that.